cli/src/cpp/usages.hpp

```diff
@@ -457,7 +457,7 @@ static const char PARTICIPANT_NAME_USAGE[] =
 
     The <profile_name> element is MANDATORY and it can be any string (whitespaces are supported if quoted).
 
-    The <participant_name> value is MANDATORY and it can be any string (whitespaces are supported if quoted).
+    SET command REQUIRES a <participant_name> value. It can be any string (whitespaces are supported if quoted).
```

Comment:
I think it should be mandatory as it was

Reply:
The `participant_name` is the argument passed to the CLI, and it is only required for `SET` command. `CLEAR` or `PRINT` does not require the name, only the `participant_profile` which is still mandatory.

Reply:
I confused it with the `<profile_name>`, you are right